decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -88,7 +88,7 @@ extra_admin_link(
     <% end %>
   </section>
 
-  <% if proposal_has_costs? && current_settings.answers_with_costs? %>
+  <% if current_settings.answers_with_costs? %>
```

Comment:
```suggestion
  <% if proposal_has_costs? && current_settings.answers_with_costs? %>
```

Reply:
Without this, you are showing the cost box without costs:

![Screenshot of the introduced bug](https://github.com/user-attachments/assets/067767c4-2dc5-4377-aa5e-fbe6d52a1926)